src/test/resources/de/tum/cit/ase/ares/integration/testuser/securitypolicies/PolicyOnePathAllowedExecuteOneCommandExecutionAllowed.yaml

```
@@ -0,0 +1,22 @@
+regardingTheSupervisedCode:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)